update-repo.py

```diff
@@ -0,0 +1,140 @@
+import subprocess
+import sys
+import os
+
+import shutil
+
+
+"""
+This script streamlines the process of copying a .yml file into each workflows directory, 
```

Comment:
good start, let's make it general that it can any add file type, not just `.yml` file throughout

Reply:
O sorry, this script is able to add any file type, I will fix the description